webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -1,143 +1,101 @@
 import PropTypes from 'prop-types';
-import React, { useState } from 'react';
-import Skeleton from 'react-loading-skeleton';
-import { ArrowIcon } from '@patternfly/react-icons';
-import ElipsisWithTooltip from 'react-ellipsis-with-tooltip';
+import React from 'react';
 import {
+  Bullseye,
   Card,
   DataList,
   DataListItem,
   DataListItemRow,
   DataListItemCells,
+  DataListWrapModifier,
   DataListCell,
+  CardActions,
   CardBody,
   CardTitle,
-  Accordion,
-  AccordionItem,
-  AccordionContent,
-  AccordionToggle,
+  CardHeader,
+  Title,
 } from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
 
 import { foremanUrl } from '../../../../foreman_tools';
 import { translate as __ } from '../../../common/I18n';
 import { useAPI } from '../../../common/hooks/API/APIHooks';
+import RelativeDateTime from '../../common/dates/RelativeDateTime';
+import SkeletonLoader from '../../common/SkeletonLoader';
+import { urlWithSearch } from '../../../common/urlHelpers';
+import { NUMBER_OF_RECORDS, BASE_URL } from './constants';
 
 const AuditCard = ({ hostName }) => {
-  const [activeAccordion, setActiveAccordion] = useState(0);
-  const onToggle = id => {
-    if (id === activeAccordion) {
-      setActiveAccordion('');
-    } else {
-      setActiveAccordion(id);
-    }
-  };
+  const encodedHostnameQuery =
+    hostName && encodeURIComponent(`host=${hostName}`);
+  const url =
+    hostName &&
+    urlWithSearch(
+      foremanUrl(BASE_URL),
+      `${encodedHostnameQuery}&per_page=${NUMBER_OF_RECORDS}`
+    );
```

Comment:
why not let URI.js handle the url building and escaping for you with
```suggestion
  const url = new URI({path: foremanUrl('/api/audits'), query: {search: `host=${hostname}`, per_page: 3}).toString()
```
Also, is there any case where there is no hostname available? should hostname be a required prop?

Reply:
should we keep using `URI.js`? it's not native, and the repo owner also mentions:
> You may not need URI.js anymore! Modern browsers provide the URL and URLSearchParams interfaces.

Reply:
Just being lazy here and adding a link for my future self: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams and https://caniuse.com/urlsearchparams.

I do agree with @tbrisker that a proper library/API is good. If browsers have it built in, that's usually the path to prefer. In this case it looks like Internet Explorer doesn't support the native API so that will probably be the thing to think about. I don't remember where we stand today with IE support.

Perhaps this should be a short term PR that uses URI.js (which I think we still use) and then a separate PR to move from URI.js to URLSearchParams?

Reply:
The fact that you *may* not need it doesn't mean we shouldn't use it - I also looked at the native API and I don't think they give a good solution for easily building or parsing paths, we have a lot of places that do that and URI.js makes it very simple to do correctly.

Reply:
Thanks,
I've added `urijs` for building the urls